conda/recipes/librapidsmpf/recipe.yaml

```diff
@@ -65,10 +65,12 @@ cache:
       - ${{ compiler("c") }}
       - ${{ compiler("cxx") }}
       - ${{ compiler("cuda") }} =${{ cuda_version }}
+      - clang==20.1.4
```

Comment:
We should use a separate CI job for clang-tidy checks, see cudf for an example. Our primary conda C++ build pipeline shouldn’t do this work, because it will make dependencies hard to manage in certain situations we’ve encountered in the past.

Reply:
Does https://github.com/rapidsai/rapidsmpf/pull/728/commits/d16e44a27ed474c99d33b62ea0d413b55d998dee satisfy your suggestion?